server/service/system/sys_auto_code.go

```diff
@@ -567,7 +567,7 @@ func (autoCodeService *AutoCodeService) GetPackage() (pkgList []system.SysAutoCo
 	return pkgList, err
 }
 
-func (AutoCodeService *AutoCodeService) DelPackage(a system.SysAutoCode) error {
+func (AutoCodeService *AutoCodeService) DelPackage(a *system.SysAutoCode) error {
 	return global.GVA_DB.Delete(a).Error
```

Comment:
```suggestion
	return global.GVA_DB.Delete(&a).Error
```
但直接delete model 可能会非预期地批量删除，建议 ` .Where(&a).Delete(&a)`  或者 `.Delete(&a,"id = ?",a.ID)` 之类的